### PR TITLE
Improve command palette

### PR DESCRIPTION
**Pull request by tjkandala**
RFC 209

- [x] - update command palette icon
- [x] - implement function to determine if only default extensions are added
- [x] - implement better empty state
- [x] - implement styles from RFC 209

### CODE REVIEW

**Comment by tjkandala** [comment]
Question: How should we introduce assets (e.g. svg) for shared components? For now, I collocated the svg with the empty command list component.

**Comment by AlicjaSuska** [comment]
Looks good so far! I've realized that we should use a 14px (body) font instead of 16px (body-lead) in the body of the empty state. Also, to make the puzzle fit without overlapping with the text, I've changed its height to 64px.

Screenshot 2020-09-10 at 15 29 56

Thank you @tjkandala !

**Comment by AlicjaSuska** [comment]
@tjkandala command palette didn't change for GitHub. Should it be addressed in this PR as well?

Also, I've added the [issue](https://github.com/sourcegraph/sourcegraph/issues/13781) regarding the extensions toggles scrolling over the command palette.

All other details we talked about are looking good :)

**Comment by lguychard** [comment]
@AlicjaSuska I don't think the browser extension was released following the merge of this branch, so the command palette wouldn't have changed on GitHub yet.

File: shared/src/commandPalette/EmptyCommandList.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.empty-command-list {
+    position: relative;
+    color: $gray-17;
```
Comment (felixfbecker) [inline shared/src/commandPalette/EmptyCommandList.scss:3]: @AlicjaSuska this is a component shared between webapp and browser extension, so this color may be off-brand on code hosts. Is it crucial to override it or could we do without it?

File: shared/src/commandPalette/EmptyCommandList.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    padding: 1rem;
+
+    &__title {
+        font-weight: 600;
```
Comment (felixfbecker) [inline shared/src/commandPalette/EmptyCommandList.scss:7]: Are you sure this override is necessary? We should generally just be using the `h*` heading style

File: shared/src/commandPalette/EmptyCommandList.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        margin-bottom: 0.5rem;
+    }
+
+    &__text {
+        line-height: 150%;
```
Comment (felixfbecker) [inline shared/src/commandPalette/EmptyCommandList.scss:12]: Similar question, is the `line-height` really different for only this element from standard body text?

**Pull request merged**